src/server/node_func.c

```diff
@@ -1899,9 +1919,9 @@ fix_indirectness(resource *presc, struct pbsnode *pnode, int doit)
 			/* setting this resource to be indirect, make serveral checks */
 
 			/* this vnode may not be a target of another indirect */
-			if (porignalrc) {
-				if (porignalrc->rs_value.at_flags & ATR_VFLAG_TARGET) {
-					if ((resc_in_err = strdup(porignalrc->rs_defin->rs_name)) == NULL)
+			if (presc_assn) {
```

Comment:
Does this need to be assigned? or can it be resources_available?

Reply:
Earlier this was checking for flags present in resources_available structure which got updated already.